dev/start.sh

```diff
@@ -19,6 +19,8 @@ cleanup() {
 set -e
 trap cleanup EXIT
 
+# cleanup from past aborted launches
+docker rm -f /dev-bindings
```

Comment:
Is the container tag really `/dev-bindings` instead of `dev-bindings`?

Reply:
The tag is `dev-bindings`.

The error we work around is:
```
The container name "/dev-bindings" is already in use by container "01c2725febfbc4af2b1acb17e46eff8ae200236ae2ea7cd46e1c855a7b84d2c7". You have to remove (or rename) that container to be able to reuse that name.
```
and the command as-is works....